### PR TITLE
chore(deps): dependencies 2023-02-27

### PR DESCRIPTION
**Pull request by millotp**
Weekly dependencies update.
Contribute to #532

### CODE REVIEW

File: clients/algoliasearch-client-javascript/package.json [hunk 1]
```diff
@@ -20,23 +20,23 @@
...
-    "nx": "14.7.5",
+    "nx": "15.8.1",
     "rollup": "2.79.1",
     "rollup-plugin-node-globals": "1.4.0",
     "rollup-plugin-terser": "7.0.2",
     "rollup-plugin-typescript2": "0.34.1",
     "semver": "7.3.8",
     "ts-node": "10.9.1",
-    "typescript": "4.8.4"
+    "typescript": "4.9.5"
```
Comment (shortcuts) [inline clients/algoliasearch-client-javascript/package.json:39]: like magically it worked??????????????

**Review by shortcuts — APPROVED** [review]
NICE!!

File: clients/algoliasearch-client-javascript/package.json (hunk 1 above)
Comment (shortcuts) [reply clients/algoliasearch-client-javascript/package.json:39]: so _I guess_ it was a yarn.lock issue right?

Comment (millotp) [reply clients/algoliasearch-client-javascript/package.json:39]: It was a lerna issue apparently, I could updgrade everything but lerna, both v5 and v6

Comment (shortcuts) [reply clients/algoliasearch-client-javascript/package.json:39]: yep I guess there might be breaking changes there, but it's fine that's the most important part

**Pull request merged**